src/pcapi/routes/native/v1/authentication.py

```diff
@@ -29,21 +36,40 @@ def signin(body: authentication.SigninRequest) -> authentication.SigninResponse:
     )
 
 
-@blueprint.native_v1.route('/refresh_access_token', methods=['POST'])
+@blueprint.native_v1.route("/refresh_access_token", methods=["POST"])
 @jwt_refresh_token_required
-@spectree_serialize(response_model=authentication.RefreshResponse, on_success_status=200, api=blueprint.api)  # type: ignore
-def refresh(body: authentication.RefreshRequest) -> authentication.RefreshResponse:
+@spectree_serialize(response_model=authentication.RefreshResponse, api=blueprint.api)  # type: ignore
+def refresh() -> authentication.RefreshResponse:
     current_user = get_jwt_identity()
-    return authentication.RefreshResponse(
-        access_token=create_access_token(identity=current_user)
-    )
+    return authentication.RefreshResponse(access_token=create_access_token(identity=current_user))
+
+
+@blueprint.native_v1.route("/password_reset_request", methods=["POST"])
+@spectree_serialize(on_success_status=204, api=blueprint.api, on_error_statuses=[503])  # type: ignore
+def password_reset_request(body: PasswordResetRequestRequest) -> None:
+    user = find_user_by_email(body.email)
+
+    if not user:
+        return
+
+    generate_reset_token(user)
+    repository.save(user)
+
+    try:
+        send_reset_password_email_to_user(user, send_raw_email)
+    except MailServiceException as mail_service_exception:
+        app.logger.error("[send_reset_password_email] Mail service failure", mail_service_exception)
```

Comment:
```suggestion
        app.logger.error(f"Email service failure when user request password reset with user.mail", mail_service_exception)
```
- Eviter de mettre de la technique dans les logs et par conséquent être plus métier. La preuve en est que Xavier t'a challengé sur le nom de la route et tu as oublié de le changer ici ;
- Le fait d'ajouter l'e-mail au message permet d'avoir un feedback rapide quand la personne va lire les logs.